rules/windows/process_creation/proc_creation_win_raspberry_robin_single_dot_ending_file.yml

```diff
@@ -0,0 +1,26 @@
+title: Raspberry Robin Dot Ending File
+id: a35c97c8-d9c4-4c89-a3e7-533dc0bcb73a
+status: experimental
+description: Detects commandline containing reference to files ending with a ".". This scheme has been seen used by raspberry-robin
+author: Nasreddine Bencherchali
+references:
+    - https://twitter.com/MichalKoczwara/status/1553634816016498688
```

Comment:
https://www.microsoft.com/en-us/security/blog/2022/10/27/raspberry-robin-worm-part-of-larger-ecosystem-facilitating-pre-ransomware-activity/

Reply:
When I pulled from master to resolve some conflicts, the ref got messed up. Thanks